department-of-veterans-affairs/va.gov-cms
# VACMS-3510: Updated Smart Date Timezone Configuration

…php time format on the hour to match php time format

## Description

See _issueid_. https://app.zenhub.com/workspaces/vagov-cms-team-5c0e7b864b5806bc2bfc2087/issues/department-of-veterans-affairs/va.gov-cms/3510

## Testing done


## Screenshots


## QA steps

- Login as Admin
- Navigate to the following Event and Situation Update and confirm the preview shows Timezone. ie: EST or EDT...etc...
  - `/outreach-and-events/events/armor-of-god-podcast`
    - [x] Wed, Nov 18 2020, 1:00am - Wed, Dec 16 2020, 11:00pm EST
  - `/va-pittsburgh-health-care/vamc-banner-alert/2020-11-02/help-limit-the-spread-of-covid-19-and-other`
    - [x] Tue, Mar 17 2020, 1:09am EDT
    - [x] Mon, Jul 27 2020, 6:59am EDT

Navigate to Smart Date config page `/admin/config/regional/smart-date/default/configure` and confirm the following:
  - [x] PHP Time Format : `g:ia T`
  - [x] PHP Time Format - on the hour: `g:ia T`


### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
